core/proto/squareup/subzero/common.proto

```diff
@@ -48,3 +48,7 @@ message EncryptedMasterSeed {
 message EncryptedPubKey {
   optional bytes encrypted_pub_key = 1;
 }
+
+message QrCodeSignature {
+  optional bytes signature = 1;
```

Comment:
nit: somewhere in the code it'd be nice to add a short comment on what the signature covers exactly (just the field `serialized_command_request`)

Reply:
done.